src/Regridder.jl

```diff
@@ -140,34 +180,42 @@ function hdwrite_regridfile_rll_to_cgll(
         @warn "Using the existing $datafile_cgll : check topology is consistent"
     end
 
-    function get_time(ds)
+    function get_coords(ds, space)
         if "time" in ds
             data_dates = Dates.DateTime.(ds["time"][:])
         elseif "date" in ds
-            data_dates = TimeManager.strdate_to_datetime.(string.(ds["date"][:]))
+            data_dates = TimeManager.strdate_to_datetime.(string.(Int.(ds["date"][:])))
         else
             @warn "No dates available in file $datafile_rll"
             data_dates = [Dates.DateTime(0)]
         end
+        if space isa Spaces.ExtrudedFiniteDifferenceSpace
```

Comment:
we could have two methods of `get_coords` that dispatch on the type of `space` instead of having an `if/else` here. But I think this way is fine too :)